passgithelper.py

```diff
@@ -22,6 +22,8 @@
 import xdg.BaseDirectory
 
 
+__version__ = "3.1.0"
```

Comment:
This won't work with the automated release model used in this repository. I don't want to maintain this version here manually. Is it needed to have it declared here or can we simply omit this?

Reply:
There was somewhere a standard (not sure which PEP), that every package should have `__version__`, but perhaps it was just for modules which should be imported by other scripts, not sure. Any way, either it should be set here and there is `version = attr: passgithelper.__version__` in `setup.cfg` using this value, or it shouldn’t be here at all, and then `version` in `setup.cfg` would be set directly. Pick your poison.

Reply:
I'd be more in favor of dropping the line from the py file and just putting it into the cfg file. However, https://github.com/jpoehnelt/semantic-release-replace-plugin would need to be added to the semantic release setup to change that line on each release. And after fixing the config, I might have to figure out permissions of the bot user.

Reply:
Isn’t d3a9a82 just what should be done: fix the generating mechanism to work appropriately?

Reply:
Oh yes, the plugin is already in place. However, the configured pattern ("from" and "to") still looks wrong.

Reply:
WHoops! Fixed.